nerfstudio/process_data/process_data_utils.py

```diff
@@ -156,6 +157,54 @@ def copy_images_list(
     return copied_image_paths
 
 
+def copy_and_upscale_polycam_depth_maps_list(polycam_depth_image_filenames: List[Path],
+                                             depth_dir: Path,
+                                             crop_border_pixels: Optional[int] = None,
+                                             verbose: bool = False) -> str:
+    """
+    Copy depth maps to working location and upscale them to match the RGB images dimensions and finally crop them
+    equally as RGB Images.
+    Args:
+        polycam_depth_image_filenames: List of Paths of images to copy to a new directory.
+        depth_dir: Path to the output directory.
+        crop_border_pixels: If not None, crops each edge by the specified number of pixels.
+        verbose: If True, print extra logging.
+    Returns:
+        A list of the copied depth maps paths.
+    """
+    copied_depth_map_paths = []
+
+    # Images should be 1-indexed for the rest of the pipeline.
+    for idx, depth_map_path in enumerate(polycam_depth_image_filenames):
+        copied_depth_map_path = depth_dir / f"frame_{idx + 1:05d}{depth_map_path.suffix}"
+        shutil.copy(depth_map_path, copied_depth_map_path)
+        copied_depth_map_paths.append(copied_depth_map_path)
+
+    # upscale them in place
+    with status(msg="[bold yellow] Upscaling depth maps...", spinner="growVertical", verbose=verbose):
+        upscale_factor = 2 ** POLYCAM_UPSCALING_TIMES
+        assert upscale_factor > 1
+        assert isinstance(upscale_factor, int)
+        depth_dir.mkdir(parents=True, exist_ok=True)
+        file_type = depth_dir.glob("frame_*").__next__().suffix
+        filename = f"frame_%05d{file_type}"
```

Comment:
It seems like this `%05d` is unreliable. #1259 has a workaround.

Reply:
Note that in the unreliable case, ffmpeg outputs a single JPEG; #1259 does have a fix, but I am double checking whether there is a ffmpeg flag that also solves it, similar to #1261, as that would be a little more efficient.

Reply:
This should also be fixed in the crop border pixels from downscale_images then too?

Reply:
> I am double checking whether there is a ffmpeg flag that also solves it, similar to https://github.com/nerfstudio-project/nerfstudio/pull/1261, as that would be a little more efficient.

No luck, #1259 still seems to be necessary.

> This should also be fixed in the crop border pixels from downscale_images then too?

I am not sure, I did not try using it with the problematic dataset (CottageMiniature) that required #1259

Reply:
I think I discovered the %05d issue, CottageMiniature includes mask images, and the RealityCapture import is treating them as non-mask images... so the image format changes from 24 bit to 1 bit, ffmpeg stops processing after first frame.  I will make a new issue for that.

Reply:
So from my understanding, there's no need to get rid of the %05d, right?

Reply:
My opinion is that you want to take what is in main (via #1259) because there is no degradation and it makes all the datasets work.

Reply:
(If you already have a list of the files to be processed, you can avoid the os.scandir to gather them that #1259 introduces for non-depth.)